pycheribuild/projects/cross/fett.py

```diff
@@ -167,6 +168,27 @@ def install(self, **kwargs):
         if not self.compiling_for_host():
             self.install_file(self.buildDir / "source/src/bvrs", self.real_install_root_dir / "var/www/cgi-bin/bvrs")
             self.install_file(self.buildDir / "source/src/bvrs.sql", self.real_install_root_dir / "share/bvrs.sql")
+            html_files = [
+                "election_official_home.html",
+                "election_official_login.html",
+                "election_official_new_voter_registration.html",
+                "election_official_query.html",
+                "index.html",
+                "index.js",
+		"jquery-3.5.1.min.js",
```

Comment:
Indentation looks a bit odd. However, I'll probably run the code formatter over the entire project soon so no real need to fix this.